modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex OpenShift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by checking the `OAuth` server provided by the platform. You can configure the following properties of Dex:
+
```

Comment:
Add the YAML code snippet under this paragraph and convert the bullet points into callouts for the YAML.